README.md

```diff
@@ -1,3 +1,8 @@
+> **🚀 You are looking at a pre-prelease of tRPC v10!**
+> See our [v10-docs.md](./.tmp/v10-docs.md) for a summary of what is changing or take a look at [the **v10 docs website**](https://alpha.trpc.io/).
```

Comment:
I'm not sure if those docs are a good source anymore vs the migration guide on the website

Reply:
Yeah let's just link the migration guide. I think we can remove that doc now?